### PR TITLE
Ssh changes io authoring

### PR DESCRIPTION
**Pull request by epankou**

### CODE REVIEW

File: src/public/js/fog/fabric-interface.js [hunk 1]
```diff
@@ -299,12 +299,45 @@ function fabricInstanceClickHandler(instanceId) {
...
+
+                checkProxyStatus(fabric.Proxy);
             }
             $('.fabric-properties-loading-overlay').css('display', 'none');
         }
     });
 }
 
+function checkProxyStatus(proxyObj) {
+    if (proxyObj == "" || proxyObj == undefined) {
```
Comment (devkudasov) [inline src/public/js/fog/fabric-interface.js]: !proxyObj

File: src/views/fog/base.ejs [hunk 2]
```diff
@@ -287,6 +287,11 @@
                     &nbsp;
                 </div>
             </div>
+            <div class="row">
```
Comment (devkudasov) [inline src/views/fog/base.ejs]: Delete this

Comment (epankou) [reply src/views/fog/base.ejs]: fixed

File: src/public/js/fog/fabric-interface.js (hunk 1 above)
Comment (epankou) [reply src/public/js/fog/fabric-interface.js]: fixed

**Pull request merged**